tests/hpc/slurm_master.pm

```diff
@@ -1,22 +1,18 @@
 # SUSE's openQA tests
 #
-# Copyright 2019-2020 SUSE LLC
+# Copyright 2019-2021 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 
 # Summary: Slurm master node
 #    This test is setting up slurm master node and runs tests depending
 #    on the slurm cluster configuration
-# Maintainer: Sebastian Chlad <sebastian.chlad@suse.com>
+# Maintainer: Sebastian Chlad <sebastian.chlad@suse.com>, Yiannis Bonatakis <ybonatakis@suse.com>
 
-use base 'hpcbase';
+use Mojo::Base 'hpcbase';
```

Comment:
I'm curious about base change, why it is needed?

Reply:
it s a small trick which inherits some modules like `strict, warnings`. So we can omit to add them to each file. https://docs.mojolicious.org/Mojo/Base

Reply:
Let me add also a link to OpenQA for future reference. https://github.com/os-autoinst/openQA/pull/4473/files Seems that the tooling team has discovered the same trick